src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/TypeWrapperTests.cs

```diff
@@ -1,151 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using Microsoft.CodeAnalysis;
-using Xunit;
-
-namespace System.Text.Json.SourceGeneration.UnitTests
```

Comment:
Do we have equivalent coverage for all applicable tests in this suite?

Reply:
Mostly they're there to validate the implementation of the type implementations, however there seems to be one test that does a lot of generic testing which I salvaged and moved to the main test suite (FWIW similar tests already exist in the main test project).